controllers/subscriptions.go

```diff
@@ -22,6 +22,12 @@ type getter func(string) []string
 
 var cache = ccache.New(ccache.Configure().MaxSize(500).ItemsToPrune(50))
 
+var bundleInfo []types.Bundle
+
+func init() {
+	bundleInfo = GetBundleInfo(config.GetConfig().Options.GetString(config.Keys.BundleInfoYaml))
+}
```

Comment:
similarly, add a comment here that the purpose of this is to read the yaml file one time at process startup.

Reply:
:rocket: